queries.sql

```diff
@@ -0,0 +1,11 @@
+/*Queries that provide answers to the questions from all projects.*/
+
+Select * from animals where name like '%mon';
+Select name from animals where date_of_birth between '20160101' and '20190101';
+Select name from animals where neutered = true and escape_attempts < 3;
+SELECT date_of_birth as "Date of Birth" FROM animals WHERE name IN ('Agumon', 'Pikachu');
+SELECT name, weight_kg FROM animals WHERE weight_kg > 10 ORDER BY weight_kg DESC;
```

Comment:
- [ ] Kindly use the correct *decimal* number asked in the requirements 👇🏽 to have the correct answer to your query. It could be possible that you will need to change 

> List name and escape attempts of animals that weigh more than 10.5kg